packages/react-core/src/components/ToggleGroup/examples/ToggleGroup.md

```diff
@@ -13,22 +13,51 @@ import ShareSquareIcon from '@patternfly/react-icons/dist/esm/icons/share-square
 
 ## Examples
 
-### Default with multiple selectable
-```ts file="./ToggleGroupDefaultMultiple.tsx"
-```
+### Single select toggle group
+
+A single select toggle group allows users to toggle between multiple items.
+
+To support single selection, use the `onChange` callback to pass the selected `<ToggleGroupItem>` to `isSelected`.
 
-### Default with single selectable
 ```ts file="./ToggleGroupDefaultSingle.tsx"
 ```
 
-### Icons
+### Multi select toggle group
+
+A multi select toggle group allows users to select multiple items at once.
+
+When a toggle item is disabled it cannot be selected. Click the "Disable all" button in the following example to see this in action.
+
+To support multiple selection, use the `onChange` callback to pass each selected `<ToggleGroupItem>` to `isSelected`.
+
+```ts file="./ToggleGroupDefaultMultiple.tsx"
+```
+
+### With icons
+
+You can use a recognizable icon as a toggle item label.
+
+To do this, pass an imported icon to the `icon` property of a `<ToggleGroupItem>`.
+
 ```ts file="./ToggleGroupIcon.tsx"
 ```
 
-### Text and icons
+### With text and icons
+
+Adding text to a toggle item with an icon helps clarify the button's function.
+
+To do this, pass a descriptive label to the `text` property of a `<ToggleGroupItem>`.
+
+When passing both `text` and `icon` properties to a `<ToggleGroupItem>`, you can also pass in `iconPosition` to determine whether the icon is rendered at the start or end of the item.
```

Comment:
Since the update that added this prop was in the v6 branch, this line should be removed for this PR, but we would want to add it to the v6 branch whenever this update gets pulled into v6 (or we could open a separate PR that adds this content to v6).

Reply:
Ah, that makes sense. Do you think I can just link that additional PR to the same issue, or is a new issue needed?